app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(params.require(:comment).permit(:text))
```

Comment:
## Optional
- [ ] I suggest that you use `the strong parameters` and remember this method should be private
```ruby
 def comment_params
    params.require(:comment).permit(:text)
  end
```